src/storage/query/QueryBoundProcessor.cpp

```diff
@@ -88,7 +87,7 @@ kvstore::ResultCode QueryBoundProcessor::processEdgeSampling(const PartitionID p
     using Sample = std::tuple<
         EdgeType, /* type */
         std::string, /* key */
-        std::unique_ptr<RowReader>, /* val */
+        RowReader, /* val */
```

Comment:
RowReader size is large. Pointer is a better choice.

Reply:
make sense